src/shell/output.c

```diff
@@ -458,7 +459,7 @@ static void shell_output_decref (struct shell_output *out,
                                  flux_msg_handler_t *mh)
 {
     if (--out->refcount == 0) {
-        flux_msg_handler_destroy (mh);
+        flux_msg_handler_stop (mh);
```

Comment:
i think this flux_msg_handler_destroy() -> flux_msg_handler_stop() is in the wrong commit?

Reply:
Good catch, it should have never been destroyed and I tried to fix up a development typo in a rebase and must have messed that up. Let me see if I can fix it up. Thanks!

Reply:
Ah, yeah it was supposed to be squashed into 4d1e7f5

Reply:
Ok, I think I've fixed that.